engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/AncientDataMotionStrategy.java

```diff
@@ -126,8 +126,8 @@ protected boolean needCacheStorage(DataObject srcData, DataObject destData) {
         if (destStoreTO instanceof NfsTO || destStoreTO.getRole() == DataStoreRole.ImageCache) {
             return false;
         }
-        if (s_logger.isDebugEnabled()) {
-            s_logger.debug("needCacheStorage true, dest at " + destTO.getPath() + " dest role " + destStoreTO.getRole().toString() + srcTO.getPath() + " src role " +
+        if (s_logger.isTraceEnabled()) {
```

Comment:
I'm sorry, @rg9975, I had to be more specific with my previous review. Can you return the log level to `debug` on the whole file? 
For maintainers, it will be more accessible when troubleshooting especially on production env.